src/test/unit/engine/CppRegionTest.cpp

```diff
@@ -457,15 +453,16 @@ TEST(CppRegionTest, realmain) {
 
 
 // TODO: This test was disabled mostly because we cannot do memLeak tests within gtest.
-/******
-TEST(DISABLED_CppRegionTest, memLeak) { //FIXME this mem leak test is newly fixed, but catches error -> need to fix code
-  / *
-   * With an integer argument 'count', runs the same test N times
+TEST(CppRegionTest, memLeak) { //FIXME this mem leak test is newly fixed, but catches error -> need to fix code
+//TEST(DISABLED_CppRegionTest, memLeak) { //FIXME this mem leak test is newly fixed, but catches error -> need to fix code
+  
+  if (true) return;
```

Comment:
keep the test DISABLED_ instead of this? As we know which tests are skipped if marked DISABLED_

Reply:
Sorry, I was just testing the test above it and needed a place to put my break-point.  It was not returning from the test (I think it did a hard crash).  I probably should have just created a dummy test for that purpose.

However, the CppRegionTest.memLeak test should probably be removed because there is no way to accurately measure a memLeak this way.  It has to be a separate module that you can put some instrumentation on it and check it after all classes have been deleted.  I don't think that can be done with gtest still running.  A real memory leak test would be great but must be something separate.

Reply:
> A real memory leak test would be great but must be something separate. 

I tried with valgrind for that, so agreed. But let's keep it separate PR, DISABLE here, and rm it later?

Reply:
Agreed.